test/test_transforms_video.py

```diff
@@ -82,7 +82,7 @@ def test_center_crop_video(self):
 
         msg = "height: " + str(height) + " width: " \
             + str(width) + " oheight: " + str(oheight) + " owidth: " + str(owidth)
-        self.assertEqual(sum1.item() > 1, True, msg)
+        assert sum1.item() > 1 == True, msg
```

Comment:
`flake` test here suggests:
> `./test/test_transforms_video.py:85:32: E712 comparison to True should be 'if cond is True:' or 'if cond:'`
Do I need to change or stick with the original implementation?

Reply:
Yes we should just use `assert sum1.item() > 1, msg` :)